chat-plugins/info.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Commands
+ * Global Commands
```

Comment:
I think "Informational commands" might be better here.

Also "These are informational commands" down below.